examples/demo_app/package.json

```diff
@@ -1,20 +1,20 @@
 {
-  "name": "demo_app",
+  "name": "demo-app",
   "private": true,
   "dependencies": {
     "@rails/activestorage": "^6.1.3",
     "@rails/ujs": "^6.1.3",
     "turbolinks": "^5.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/preset-env": "^7.5.5",
-    "babel-loader": "^8.0.6",
-    "glob": "^7.1.4",
-    "webpack": "^4.39.3",
-    "webpack-assets-manifest": "^3.1.1",
-    "webpack-cli": "^3.3.7",
-    "webpack-dev-server": "^3.8.0"
+    "@babel/core": "^7.13.15",
+    "@babel/preset-env": "^7.13.15",
+    "babel-loader": "^8.2.2",
+    "glob": "^7.1.6",
+    "webpack": "^4.46.0",
+    "webpack-assets-manifest": "^4.0.5",
```

Comment:
This major version upgrade isn't necessary for rails 6.1, but it doesn't hurt to upgrade now.